tests/micro-benchmarks/TestHasEq.fst

```diff
@@ -84,6 +84,6 @@ type erasable_t =
 let test (x:erasable_t{C_erasable_t? x}) (y:erasable_t{D_erasable_t? y}) : Tot (n:int{n == 1}) =
   if x = y then 0 else 1  //this would extract to if () = () then 0 else 1 if we allowed equality on erasable_t
 
-[@@erasable expect_failure]
+[@@erasable; expect_failure]
```

Comment:
wow. But I'm confused.. was this not under an `expect_failure` before? Hence succeeding?

Reply:
Yeah, that's pretty confusing, I was quite puzzled as well!
But that's just that neither the `erasable` nor the `expect_failure` attributes were kicking in here. Thus, the failure regarding the inductive not being erasable was not happening, and this "non-failure" was fine since the `expect_failure` attribute was disabled as well...
So it's a bit of a double negation thing, both `[@@erasable; expect_failure]` and `[@@erasable expect_failure]` typecheck here ^^

Btw, I'm thinking: this bug is quite a nice motivation for typechecking arguments! ^^